stylesheets/views/_register.scss

```diff
@@ -0,0 +1,33 @@
+.register {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    height: 85vh;
+  
+    @media screen and (min-width: $breakpoint-tablet) {
+      background-color: #444444;

```

Comment:
I notice that another file adds sass variables for color, but in this one you don't use them.

Reply:
Done